Library/Formula/catimg.rb

```diff
@@ -0,0 +1,21 @@
+class Catimg < Formula
+  desc "Insanely fast image printing in your terminal"
+  homepage "https://github.com/posva/catimg"
+  url "https://github.com/posva/catimg/archive/v2.2.1.tar.gz"
+  sha256 "eb76d3baeb5e7382d8839d7d4351794166c0b6e8d777ffe0087a3401f907e991"
+
+  head do
+    url "https://github.com/posva/catimg.git"
```

Comment:
If you don't need any extra dependencies or anything here you're good to just tack `head "https://example.git"` just under the `sha256` rather than in its own block.